src/civis/resources/_resources.py

```diff
@@ -17,20 +17,16 @@
 API_VERSIONS = frozenset({"1.0"})
 
 # civis_api_spec.json can be updated
-# by running the tools/update_civis_api_spec_json.py script.
+# by running the tools/update_civis_api_spec.py script.
 API_SPEC_PATH = os.path.join(
     os.path.dirname(os.path.realpath(__file__)),
     "civis_api_spec.json",
 )
 with open(API_SPEC_PATH) as f:
     API_SPEC = json.load(f, object_pairs_hook=OrderedDict)
-BASE_RESOURCES_V1 = sorted(
```

Comment:
`BASE_RESOURCES_V1` needs to be removed, as it controlled what API resources would be included / excluded by `civis.resources.generate_classes_maybe_cached`. The resources included in `BASE_RESOURCES_V1` were based on the versioned `civis_api_spec.json`, which would directly interfere with the goal of the new script `tools/check_if_civis_api_spec_has_updated.py`.